pyxem/signals/beam_shift.py

```diff
@@ -81,6 +87,23 @@ def get_linear_plane(self, mask=None, fit_corners=None):
         fit_corners : float, optional
             Make a mask so that the planes are fitted to the corners of the
             signal. This mush be set with a number, like 0.05 (5%) or 0.10 (10%).
+        initial_values : array of floats, optional
+            Initial guess for the plane parameters. Useful to vary if the plane fitting
+            does not give desirable results. The parameters are as follows:
+            [d/dx, d/dy, x_0, d/dx, d/dy, y_0]
+            where the first three entries are for the x-shift, being in order the
+            step in x, step in y and the initial value at (0, 0). Similarly for the
+            last three entries for the y-shift. Currently only implemented for the
```

Comment:
Can you maybe reword this?  I'm not exactly sure what these parameters are/ what they do.

Reply:
Yes, these parameters are a bit hard to briefly introduce, especially as we have to differentiate between an x-shift and an x-position. I will try to reword it.